README.md

```diff
@@ -34,81 +35,96 @@ RAM is an app to create an appointment to test drive a motorcyle.
 </details>
 
 <details>
-  <summary>Server</summary>
+  <summary>Frontend</summary>
   <ul>
-    <li>Ruby-on-Rails</li>
+    <li>React</li>
   </ul>
 </details>
 
 <details>
-<summary>Database</summary>
+  <summary>Backend</summary>
   <ul>
-    <li>Postgress</li>
+    <li>Ruby on Rails</li>
   </ul>
 </details>
 
 ### Key Features <a name="key-features"></a>
 
 - Add appointment for a motorbike
 - Delete appointment for a motorbike
+- Create and Delete motorbikes
 
 
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<p align="right">(<a href="#table-of-contents">back to top</a>)</p>
 
 
-## 💻 Getting Started <a name="getting-started"></a>
+## Getting Started <a name="getting-started"></a>
 
 To get a local copy up and running, follow these steps.
 
-
 ### Prerequisites
 
-In order to run this project you need: any web-browser.
+In order to run this project you need: 
+- Any web-browser.
+- Latest version of Ruby on Rails
 
 ### Setup
 
 This API is consumed by its front-end counterpart [front-end](https://github.com/c00p75/appointment-app-frontend)
 
-
-Clone this [repo](https://github.com/c00p75/appointment-app-frontend.git) to your desired folder.
+Clone this [repo](https://github.com/c00p75/appointment-app-backend.git) to your desired folder.
 
 ```
-    git clone https://github.com/c00p75/appointment-app-frontend.git
+    git clone https://github.com/c00p75/appointment-app-backend.git
 ```
 
 ### Install
 
 Run from a terminal
 
 ```
-    cd appointment-app-frontend
+    cd appointment-app-backend
     bundle install
 ```
 
+### Database
+**To connect the project to the database:-**
+```
+    rails db:create
+    rails db:migrate
+```
+
 ### Usage
 
 **To run the project follow the following instruction:-**
 
 ```
-    rails db:migrate
-    rails s
+    rails server
 ```
 
-### Run tests
+### Run tests (using rspec)
 
-To run tests, using RSpec
+To run tests
 
 ```
-  rspec spec
+  bundle exec rspec
 ```
 
```

Comment:
- [ ] When I run the tests according to your instructions, this happens:

![image](https://github.com/c00p75/appointment-app-backend/assets/57421823/a9acae48-2f7d-40b6-92d9-ce489969878e)

This is because:

1. Some of your tests are rswag tests.  rspec won't run them properly and they will fail.

Solution: Please be more specific with your commands.  This is a way to be more precise:

```bash
rspec spec/controllers
rspec spec/models
```

2. The controller tests have 5 failures:

![image](https://github.com/c00p75/appointment-app-backend/assets/57421823/f85a3cfb-c10f-48df-903a-e10ba62224dd)

Kindly fix your controller tests so they pass.  If there are special instructions to run your tests properly, (like a secret token, a `.env` file required, etc.) please add those instructions to your readme file.

That way, all your tests will pass for all your users. 👍

Reply:
fixed, thanks!